app/views/workspaces/viewer.html.erb

```diff
@@ -0,0 +1,35 @@
+<% unless @embed %>
+  <div class="d-flex justify-content-between">
+    <%= link_to '<i class="bi bi-arrow-left"></i> Back'.html_safe, @workspace, class: 'btn btn-link' %>
+    <% if can? :update, @workspace %>
+      <%= react_component("EditInPlace", { field: 'title', value: @workspace.title, csrfToken: form_authenticity_token, url: url_for(@workspace) }, tag: 'h1') %>
+
+      <%= form_with(model: @workspace, class: 'me-2') do |form| %>
+        <%= form.hidden_field :state %>
+        <%= form.submit 'Save workspace', class: 'btn btn-primary' %>
+      <% end %>
+    <% else %>
+      <h1><%= @workspace.title %></h1>
+      <span></span>
+    <% end %>
+  </div>
+<% end %>
+
+<div class="<%= @embed ? 'embedded-mirador-container' : 'mirador-container mx-2' %>">
+  <% viewer_props = if can? :update, @workspace
```

Comment:
Do we want to do this off of `can? :update`? I ask because the logged in user will get a different view at `/embed` rather than what others would see.